### DISCUSSION TITLE
Completion file for command wrapper without arguments

### DISCUSSION THREAD

**Discussion by vulns-are-features-too** [marlonrichert/zsh-autocomplete#403] [Q&A] [answered]
I use [devour](https://github.com/salman-abedin/devour) which works as a wrapper around other commands. Running `devour mpv ...` would just run `mpv` normally but with the extra effect of hiding the terminal in which the command was run. `devour` has no other arguments, and `devour mpv <TAB>` should suggest flags & args for `mpv` like it was run without `devour`.

I can't quite figure out how to get such suggestion with this autocomplete. How can I get such behavior?

**Comment by marlonrichert** [comment, marked as answer]
Completions are generated by Zsh's completion system, not by the this plugin. Try asking your question on the [Zsh users mailing list](https://zsh.sourceforge.io/Arc/mlist.html).

**Reply by vulns-are-features-too** [reply]
I've looked around for completions but many of them assume `compinit` is called somewhere in `.zshrc`. However, your [installation guide](https://github.com/marlonrichert/zsh-autocomplete#manual-installation) says to **remove** all `compinit` calls from `.zshrc`, and a quick `ripgrep` on the repo shows that `compinit` isn't used anywhere in your code. How should I handle this issue?

**Reply by pcjmfranken** [reply]
https://github.com/marlonrichert/zsh-autocomplete/blob/main/scripts/.autocomplete.compinit

But it is, from its very own file even 😉

**Reply by vulns-are-features-too** [reply]
OK it was ignored by both `fd` and `rg` because it's hidden.
